[pmdk] build failure during installation

**Host Environment**
- OS: Ubuntu 20.04.1

**To Reproduce**
`./vcpkg install pmdk:x64-linux`

**Failure Logs**
`Computing installation plan...
The following packages will be built and installed:
    pmdk[core]:x64-linux -> 1.9
Detecting compiler hash for triplet x64-linux...
Could not locate cached archive: /home/ethan/.cache/vcpkg/archives/f9/f9f19ffb279a70921d71ef36248180cb7b9a1e2a.zip
Starting package 1/1: pmdk:x64-linux
Building package pmdk[core]:x64-linux...
-- Note: pmdk only supports dynamic library linkage. Building dynamic library.
-- Using cached /home/ethan/vcpkg/downloads/pmem-pmdk-1926ffb8f3f5f0617b3b3ed32029d437c272f187.tar.gz
-- Cleaning sources at /home/ethan/vcpkg/buildtrees/pmdk/src/37c272f187-acb9c26f73.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/ethan/vcpkg/downloads/pmem-pmdk-1926ffb8f3f5f0617b3b3ed32029d437c272f187.tar.gz
-- Applying patch remove-non-ascii-character.patch
-- Using source at /home/ethan/vcpkg/buildtrees/pmdk/src/37c272f187-acb9c26f73.clean
-- Building /home/ethan/vcpkg/buildtrees/pmdk/src/37c272f187-acb9c26f73.clean/src/PMDK.sln for Release
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:108 (message):
    Command failed: msbuild /home/ethan/vcpkg/buildtrees/pmdk/src/37c272f187-acb9c26f73.clean/src/PMDK.sln /p:Configuration=Release /p:SRCVERSION=1.9 "/t:Solution Items\libpmem,Solution Items\libpmemlog,Solution Items\libpmemblk,Solution Items\libpmemobj,Solution Items\libpmempool,Solution Items\Tools\pmempool" /p:Platform=x64 /p:PlatformToolset=external /p:VCPkgLocalAppDataDisabled=true /p:UseIntelMKL=No /p:WindowsTargetPlatformVersion= /p:VcpkgManifestInstall=false /m
    Working Directory: /home/ethan/vcpkg/buildtrees/pmdk/x64-linux-rel
    Error code: No such file or directory
    See logs for more information:

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_msbuild.cmake:123 (vcpkg_execute_required_process)
  ports/pmdk/portfile.cmake:18 (vcpkg_build_msbuild)
  scripts/ports.cmake:128 (include)


Error: Building package pmdk:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: pmdk:x64-linux
  Vcpkg version: 2021-01-13-unknownhash`

I see that #11096 had a similar problem and the issue was closed as it had been fixed but it appears that the same issue has returned. I've tried updating vcpkg and doing a fresh install but neither have changed the issue.

--- a/ports/pmdk/vcpkg.json
+++ b/ports/pmdk/vcpkg.json
@@ -1,11 +1,11 @@
 {
   "name": "pmdk",
   "version": "1.12.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Persistent Memory Development Kit",
   "homepage": "https://github.com/pmem/pmdk",
   "license": "BSD-3-Clause",
-  "supports": "!(arm | x86)",
+  "supports": "!(linux |arm | x86)",
   "dependencies": [
     "getopt",
     {

--- a/ports/pmdk/vcpkg.json
+++ b/ports/pmdk/vcpkg.json
@@ -5,7 +5,7 @@
   "description": "Persistent Memory Development Kit",
   "homepage": "https://github.com/pmem/pmdk",
   "license": "BSD-3-Clause",
-  "supports": "!(linux |arm | x86)",
+  "supports": "!(linux | arm | x86)",
   "dependencies": [
     "getopt",
     {

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6458,7 +6458,7 @@
     },
     "pmdk": {
       "baseline": "1.12.0",
-      "port-version": 1
+      "port-version": 2
     },
     "pngpp": {
       "baseline": "0.2.10",

--- a/versions/p-/pmdk.json
+++ b/versions/p-/pmdk.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "b208389a2bc32898f0f4d13c5f222015ce9bd072",
+      "version": "1.12.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "5dc7f0e5a4d6348ac49617010105730775580f78",
       "version": "1.12.0",
       "port-version": 1

